tests/tests_assemble_workflow/data/opensearch-dashboards-build-windows-1.3.0.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Why are we adding windows changes in this PR? I think its better to keep the scope of this PR limited to rpm changes

Reply:
This is not to add windows changes, rather adding distribution related tests.
The PR itself is also not adding rpm, but adding assemble based on distribution not extension.
Therefore, the windows distribution manifest provide a great scenario of `zip` aside from the default `tar` for us to test.